src/App.jsx

```diff
@@ -49,7 +49,10 @@ export function App() {
 						element={<Home data={lists} setListPath={setListPath} />}
 					/>
 					<Route path="/list" element={<List data={data} userId={userId} />} />
-					<Route path="/manage-list" element={<ManageList userId={userId} />} />
+					<Route
+						path="/manage-list"
+						element={<ManageList userId={userId} data={data} />}
+					/>
```

Comment:
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking ):** Total nitpick here :) I'd encourage you to avoid using the word `data` - a more descriptive name like `list` is ideal

Reply:
And I know data is what's used by default in this file. If you don't want to change it here, you can just change it at the `ManageList` level like so:

`<ManageList userId={userId} list={data}`